scripts/zika.py

```diff
@@ -165,20 +165,20 @@ def format_worklist(df, deck, split_transfers = False):
     """
     - Add columns in Mosquito-intepretable format
     - Resolve multi-transfers
-    - Sort by dest col, dest row, buffer, sample
+    - Sort by dst col, dst row, buffer, sample
     """
 
     # Add columns for plate positions
-    df["src_pos"] = df["source_fc"].apply(lambda x: deck[x])
```

Comment:
Just to double check: is it also needed to change the variable name in line 237?